### PR TITLE
Pyu/afp 174 establish ci pipeline

### PR DESCRIPTION
**Pull request by pyupew**

### CODE REVIEW

File: packages/website/pages-list.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* pages-list.json is only generated upon building the website, and so will not exist
+   during linting. This file exists so there is a single place where pages-list.json is
+   imported so the eslint rule only needs to be disabled here, as opposed to every file
+   pages-list.json is used.
+*/
+
+/* eslint-disable import/no-unresolved */
```
Comment (dpisani-atl) [inline packages/website/pages-list.js]: FYI you can also do `// eslint-disable-next-line import/no-unresolved`

File: packages/website/pages/docs.js [hunk 2]
```diff
@@ -10,7 +10,9 @@ import Link from 'next/link';
 import titleCase from 'title-case';
 
 import Page, { Section, Title } from '../components/page';
-import pageInfo from '../data/pages-list.json';
+/* eslint-disable import/no-unresolved */
```
Comment (dpisani-atl) [inline packages/website/pages/docs.js]: is this still necessary?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    # fallback to using the latest cache if no exact match is found
+    - v1-dependencies-
+
+save-cache: &save-cache
+  paths:
+    - node_modules
+    - ~/.cache
+  key: v1-dependencies-{{ checksum "package.json" }}
+
+install-bolt: &install-bolt
```
Comment (dpisani-atl) [inline .circleci/config.yml]: Is this being used?

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+  lint:
+    <<: *job-setup
+    steps:
+      - checkout
+
+      - restore_cache: *restore-cache
+
+      - run:
+          name: Install Bolt
```
Comment (dpisani-atl) [inline .circleci/config.yml]: Is this not covered by the `install` job?

Comment (pyupew) [reply .circleci/config.yml]: The caching wasn't working before so I was installing dependencies in each job independently. I'll try removing it now and see what happens

File: packages/website/components/page-templates/package-home.test.js [hunk 5]
```diff
@@ -3,6 +3,9 @@ import titleCase from 'title-case';
 import PackageHomeWrapper, { Description } from './package-home';
 
 jest.mock('../navigation/package-nav-content', () => () => <div />);
+/* eslint-disable react/prop-types */
```
Comment (Noviny) [inline packages/website/components/page-templates/package-home.test.js]: Not blocking, just echoing:

`/* eslint-disable-next-line react/prop-types */` is slightly neater.

**Pull request merged**